Razor.sln

```diff
@@ -16,6 +16,7 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{F8C12DD6-659D-405A-AA27-FB22AD92A010}"
 	ProjectSection(SolutionItems) = preProject
 		.editorconfig = .editorconfig
+		build\dependencies.props = build\dependencies.props
```

Comment:
How come we didn't need this before?

Reply:
Still don't need it. It's just nice to have. This makes the dependencies.props file show up in the `Solution Items`

Reply:
Oh duh. Just realized it